TrafficLight.py

```diff
@@ -0,0 +1,111 @@
+""" Represents the state of a traffic flow at a single light, a matrix made of rows that each
```

Comment:
Python convention is to put this under the class declaration. Documentation is informative, but a tad confusing when describing arguments. Try to follow the same convention as later explicitly stating parameter and type in a list